data/RI/incentives.json

```diff
@@ -245,6 +245,54 @@
       "es": "Un reembolso adicional de $1,500 por un vehículo eléctrico usado para los calificadores de LIHEAP. Por orden de llegada."
     }
   },
+  {
+    "id": "RI-13",
```

Comment:
I tried to add fields `program_status` and `other_restrictions`, but they were causing test failures, I think because they're not included [here](https://github.com/rewiringamerica/api.rewiringamerica.org/blob/main/src/data/state_incentives.ts#L171-L174). I ended up removing those fields and it sounds like from slack, program_status isn't used by the calculator anyway (is other_restrictions?) so maybe this is fine as-is.